plugins/out_s3/s3.c

```diff
@@ -1073,6 +1073,18 @@ static int upload_data(struct flb_s3 *ctx, struct s3_file *chunk,
         if (chunk) {
             s3_store_file_unlock(chunk);
             chunk->failures += 1;
+            char create_time_string[80];
+            strftime(create_time_string, 80, "%Y-%m-%d %H:%M:%S", localtime(&(create_time)));
+            if(chunk->failures <= ctx->ins->retry_limit){
+                flb_plg_warn(ctx->ins,
+                             "failed to flush chunk tag= %s, create_time= %s"
+                             "retry issued: input=forward.1 > output=s3.0 (out_id=0)",
```

Comment:
This bit `input=forward.1 > output=s3.0 (out_id=0)` was just an example from the doc. We need to replace these with the actual input and output IDs.

Remember I showed you how to get those. The output instance is the `ins` var you already know, and the input instance `i_ins` is sent in the flush callback that I showed you.